### PR TITLE
Magic Survivalists can have spellbooks count if they did not use them

### LINKED ISSUE #40857: Steal Magical Artifacts Objective
Issue reported from  Round ID: 95272 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 512

Objective seems bugged? I had a scrying orb, healing wand, 2 blindness tomes, 1 instant recall tome, and 2 tomes of smoke, but didn't get greentext for collecting artifacts.

### PR DESCRIPTION
**Pull request by tralezab**
:cl:
tweak: spellbooks now count towards the survivalist objective if they were not used.
/:cl:

fixes #40857

It feels like shit if you don't know spellbooks don't count and you just lose out of nowhere. It also doesn't make sense for them to count if they still have magic powers.

### CODE REVIEW

**Comment by coiax** [comment]
Who doesn't use a spellbook that they find

**Comment by tralezab** [comment]
It would be pretty interesting to see a magic survivalist win by not using their starting spellbook. But anyway, it's up to them

**Comment by ATH1909** [comment]
It should also count spells that you know (or maybe just the ones that you learn after becoming an amateur magician?), IMO, but that'd be hard to code (since some things that aren't spells fluff-wise are technically spells in the code).

**Comment by 4dplanner** [comment]
There are good reasons to not use spellbooks, for instance getting them on the shuttle and yackety saxing until centcomm

**Comment by tralezab** [comment]
Well, I just wanted to make clear when you are getting a point counted for an item or not.

File: code/modules/antagonists/survivalist/survivalist.dm [hunk 1]
```diff
@@ -31,8 +31,12 @@
 	name = "Amateur Magician"
 	greet_message = "Grow your newfound talent! Grab as many magical artefacts as possible, by any means necessary. Kill anyone who gets in your way."
 
+/datum/antagonist/survivalist/greet()
```
Comment (Cyberboss) [inline code/modules/antagonists/survivalist/survivalist.dm]: Wrong type?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/survivalist/survivalist.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/antagonists/survivalist/survivalist.dm]: my bad

**Pull request merged**